File: guanabara-python/src/lista02/exercicio24.py
quantidade_km_percorridos = float(input("Insira a distância a ser percorrida (em km): "))

if quantidade_km_percorridos <= 200:
    preco_viagem = quantidade_km_percorridos * 0.50
else:
    preco_viagem = quantidade_km_percorridos * 0.50

print(f"Preço da viagem: R$ {preco_viagem}")

File: guanabara-c/lista02/exercicio24.c
#include <stdio.h>

int main(){

    int km_percorridos;
    float preco_passagem;

    printf("Insira a distância da viagem em km: ");
    scanf("%d", &km_percorridos);
    
    if(km_percorridos <= 200){
        preco_passagem = km_percorridos * 0.50;
    }
    else{
        preco_passagem = km_percorridos * 0.45;
    }

    printf("Preço da passagem: R$ %.2f", preco_passagem);

    return 0;
}
